src/LibraryViewExtensionWebView2/Views/LibraryView.xaml.cs

```diff
@@ -0,0 +1,20 @@
+﻿
+using System.Windows.Controls;
+
+using Dynamo.LibraryViewExtensionWebView2.ViewModels;
```

Comment:
Let's remove the empty lines

Reply:
@QilongTang fixed in the next commit: https://github.com/DynamoDS/Dynamo/pull/13330/commits/dbbfffde0f9cbb0b64f134820ca012abfb1f12ad